modules/core/src/main/scala/com/siriusxm/snapshot4s/Repr.scala

```diff
@@ -47,12 +49,12 @@ object Repr extends ReprForAdt {
   implicit val reprForFloat: Repr[Float]     = fromPprint
```

Comment:
We could use `MultiLineRepr` for these and remove `pprint` entirely, however this can be done in a separate PR.